changelog/805.feature.rst

```diff
@@ -0,0 +1 @@
+relativistic_energy() is a function that uses the establised Lorentz_factor() function to calculate the relativistic energy of a particle in Joules from the mass of the particle in kilograms and the speed in meters per second.
```

Comment:
```suggestion
Create :func:`~plasmapy.formulary.relativity.relativistic_energy` function, which uses the established :func:`~plamsapy.formulary.relativity.Lorentz_factor` function to aid in the calculation of the relativistic energy of an object.
```

The [`:func:`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html?highlight=%3Afunc%3A#role-py-func) here is a specialize "role" used by `sphinx` (which builds our documentation) to automatically cross-reference the functions in our documentation.  The `~` indicates to `sphinx` to only display the last part in the dotted reference (i.e. `relativistic_energy` instead of `plasmapy.formulary.relativity.relativistic_energy`).

Reply:
So that's what it means! Thanks, TIL!